src/Dialogs/Airspace/AirspaceList.cpp

```diff
@@ -160,13 +161,21 @@ static constexpr StaticEnumChoice type_filter_list[] = {
   { NOGLIDER, _T("No gliders") },
   { CTR, _T("CTR") },
   { WAVE, _T("Wave") },
+  { AATASK, _T("Task Area") },
   { CLASSE, _T("Class E") },
   { CLASSF, _T("Class F") },
   { TMZ, _T("TMZ") },
+  { CLASSG, _T("Class G") },
   { MATZ, _T("MATZ") },
+  { RMZ, _T("RMZ") },
   nullptr
 };
 
+static_assert(ARRAY_SIZE(type_filter_list) ==
+              (size_t)AirspaceClass::AIRSPACECLASSCOUNT + 2,
+              "number of airspace class names does not match number of "
```

Comment:
Why add 2 to the airspace count? It would be more logical to subtract ARRAY_SIZE(type_filter_list) and explain why.

Reply:
I thought something like this:

```
static_assert(ARRAY_SIZE(type_filter_list) - 2 ==  // WILDCARD and nullptr do not belong to airspace
              (size_t)AirspaceClass::AIRSPACECLASSCOUNT,
              "number of airspace class names does not match number of "
              "airspace classes");
```

Reply:
Done, formatted the comment above, as that would execeed char count per line.